app/src/main/java/com/sdp/movemeet/backend/firebase/firestore/FirestoreUserManager.java

```diff
@@ -17,11 +18,28 @@
     private final FirebaseFirestore db;
     private final String collection;
 
+    /**
+     * Creates a new FirestoreManager capable of performing backend storage operations
+     * on the User class.
+     * @param db the FirebaseFirestore instance serving as a backend
+     * @param collection the Firestore collection in which to operate
+     * @param serializer a BackendSerializer capable of (de)serializing Users
+     */
     public FirestoreUserManager(FirebaseFirestore db, String collection, BackendSerializer<User> serializer) {
```

Comment:
I think you should add basic tests for those classes, to higher up the coverage.

Reply:
I believe that this class was import from kepler branch and it seems to me that he said he was going to do the tests.